### PR TITLE
[xOWL] deployment

### PR DESCRIPTION
**Pull request by fleupold**
This PR adds a migration script, deployment instructions and the initial deployment info of the xOWL token which has been building up over the last couple of PRs (if changes are requested above, I will simply redeploy and update this PR)

### Test Plan

```
export PK=...
yarn migrate --network xdai -f 6
```

Follow the instruction to verify contract.

I also tested that I can use this contract to setup the other components required for an OWL bridge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +115,24 @@ You can use [verify-on-etherscan](https://www.npmjs.com/package/verify-on-ethers
...
 ```
 
+## Verify contract on xDAI
+
+Flatten the smart contract:
+```bash
+npx truffle-flattener contracts/4/BridgedTokenOWL.sol > build/BridgedTokenOWL-BlockScout.sol
+```
+
+Go to Blockscout validation page:
```
Comment (rmeissner) [inline README.md:125]: Would be nice to have the validation also on [sourcify](https://verification.komputing.org/)

**Review by rmeissner — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (fleupold) [reply README.md:125]: As far as I can tell sourcify doesn't yet support the xDAI chain.

Comment (rmeissner) [reply README.md:125]: ohhh true ... my fault ... sorry 😬

File: migrations/6_deploy_bridged_token_owl_on_xdai.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* global artifacts, web3 */
+
+const BridgedTokenOWL = artifacts.require('BridgedTokenOWL')
+
+module.exports = async function (deployer, network, accounts) {
+  if (process.env.DEPLOY_OWL_ONLY) {
```
Comment (nlordell) [inline migrations/6_deploy_bridged_token_owl_on_xdai.js:6]: Maybe this deployment script should only deploy if `networkId === 100`, or else couldn't `BridgedTokenOWL` potentially get accidentally deployed on Mainnet as well?

**Review by nlordell — APPROVED** [review]

**Pull request merged**